src/edu/csus/ecs/pc2/core/TeamStatus.java

```diff
@@ -0,0 +1,32 @@
+package edu.csus.ecs.pc2.core;
+
+/**
+ * Team contact status.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public enum TeamStatus {
+    /**
+     * Team has not logged in.
+     */
+    NO_TEAM_CONTACT,
+    /**
+     * Team has logged in only.
+     */
+    TEAM_HAS_LOGGED_IN,
+    
+    /**
+     * Team has submitted runs only.
+     */
+    TEAM_HAS_SUBMITTED_RUNS_ONLY,
+    
+    /**
+     * Team has submitted clars only.
+     */
+    TEAM_HAS_SUBMITTED_CLARS_ONLY,
+    /**
+     * Team has submitted runs and clars
+     */
+    TEAM_HAS_SUBMITTED_RUNS_AND_CLARS,
```

Comment:
I'm surprised to find that the javac compiler accepts the trailing comma after this last enum element...

Reply:
Me too, turns out that array initialization also allows a trailing comma.   This works too: String [] arrayOne = { "abc"," def", };